src/PositionManager.sol

```diff
@@ -110,24 +110,20 @@ contract PositionManager is
     using SafeTransferLib for *;
     using CurrencyLibrary for Currency;
     using PoolIdLibrary for PoolKey;
-    using PositionConfigLibrary for PositionConfig;
     using StateLibrary for IPoolManager;
     using TransientStateLibrary for IPoolManager;
     using SafeCast for uint256;
     using SafeCast for int256;
     using CalldataDecoder for bytes;
     using SlippageCheckLibrary for BalanceDelta;
-    using PositionConfigIdLibrary for PositionConfigId;
+    using PoolKeyChecker for PoolKey;
+    using PositionInfoLibrary for PackedPositionInfo;
 
     /// @dev The ID of the next token that will be minted. Skips 0
     uint256 public nextTokenId = 1;
 
-    mapping(uint256 tokenId => PositionConfigId configId) internal positionConfigs;
-
-    /// @notice an internal getter for PositionConfigId to be used by Notifier
-    function _positionConfigs(uint256 tokenId) internal view override returns (PositionConfigId storage) {
-        return positionConfigs[tokenId];
-    }
+    mapping(uint256 tokenId => PackedPositionInfo info) public positionInfo;
```

Comment:
now that its packed maybe worth having this internal and exposing a getter that returns the elements individually?

Reply:
hm I dont feel strongly about it. For offchain, a simple sdk helper can dissect it. and onchain someone can just import the PositionInfoLibrary?

But if you prefer I can add